tests/unit/qunit/test-loader-test.js

```diff
@@ -19,20 +19,19 @@ module('Unit | test-loader', {
       'test-4-test': true,
       'test-4-test.jshint': true,
     };
-
-    this.originalURLParams = TestLoader._urlParams;
+    this.testLoader = new EmberExamTestLoader();
+    this.originalURLParams = EmberExamTestLoader._urlParams;
   },
 
   afterEach() {
-    TestLoader._urlParams = this.originalURLParams;
+    EmberExamTestLoader._urlParams = this.originalURLParams;
     window.require = this.originalRequire;
   }
 });
 
 test('loads all test modules by default', function(assert) {
-  TestLoader._urlParams = {};
-
-  TestLoader.load();
+  this.testLoader._urlParams = {};
```

Comment:
`_` prefixed variables are considered private, mutating them external in tests is very unexpected. It is breaking encapsulation.  If we must changes these arguments on every test, lets simply instantiate a new `EmberExamTestLoader` class for each test.

```js
const testLoader = new EmberExamTestLoader({} /* urlParams */);

// or maybe:
const testLoader = new EmberExamTestLoader({ urlParams });
```

and then no cleanup is required, as no global pollution is possible.